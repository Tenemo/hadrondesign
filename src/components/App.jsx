import React from 'react';
import { hot } from 'react-hot-loader';
import Routes from './Routes';
import Header from './common/Header';

class App extends React.Component {
    render() {
        return (
            <React.Fragment>
                <Header />
                <div className="container-fluid main-container">
                    <Routes />
                </div>
            </React.Fragment>
        );
    }
}

export default hot(module)(App);