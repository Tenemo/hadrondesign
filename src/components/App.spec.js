import React from 'react';
import App from './App';

describe('App', () => {
    // it('should have an h1 About header', () => {
    // });
    it('matches snapshot', () => {
        // const wrapper = shallow(
        //     <App />
        // );
        // expect(wrapper).toMatchSnapshot();
    });
});