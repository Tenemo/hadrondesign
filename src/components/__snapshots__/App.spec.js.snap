// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches snapshot 1`] = `
<React.Fragment>
  <Header />
  <div
    className="container-fluid main-container"
  >
    <Routes />
  </div>
</React.Fragment>
`;
